lib/node_modules/@stdlib/blas/ext/base/dapx/docs/repl.txt

```diff
@@ -9,7 +9,7 @@
     Indexing is relative to the first index. To introduce an offset, use typed
```

Comment:
The above should read "which elements in the strided array are accessed at runtime". And you'll want to rewrap the line.